### ISSUE TITLE
GTK4

### ISSUE THREAD

**Issue by musiclover382** [eneshecan/whatsapp-for-linux#239]
**Describe the solution you'd like**
Like any other app, WFL could benefit from a GTK4 (and optionally libadwaita) port (like improved text rendering, better scrolling, aesthetics for libadwaita)

Definitely not a high-priority feature though 😬️👍️

**Comment by eneshecan** [comment]
Yeah, I've been thinking about this for a long time but didn't really have time to do it.

**Comment by cesctiger** [comment]
> **Describe the solution you'd like**
> 
> Like any other app, WFL could benefit from a GTK4 (and optionally libadwaita) port (like improved text rendering, better scrolling, aesthetics for libadwaita)
> 
> 
> 
> Definitely not a high-priority feature though 😬️👍️
